PurchasesTests/Purchasing/OfferingsTests.swift

```diff
@@ -187,26 +184,38 @@ class OfferingsTests: XCTestCase {
         testPackageType(packageType: PackageType.unknown)
     }
 
-    func testNoOfferings() {
+    func testOfferingsIsNilIfNoOfferingCanBeCreated() throws {
         let data = [
             "offerings": [],
             "current_offering_id": nil
         ]
-        let offerings = offeringsFactory.createOfferings(withProducts: [:], data: data as [String : Any])
+        let maybeOfferings = offeringsFactory.createOfferings(withProducts: [:], data: data as [String : Any])
 
-        expect(offerings).toNot(beNil())
-        expect(offerings!.current).to(beNil())
+        expect(maybeOfferings).to(beNil())
     }
 
-    func testCurrentOfferingWithBrokenProduct() {
+    func testCurrentOfferingWithBrokenProductReturnsNilForCurrentOfferingButContainsOtherOfferings() throws {
```

Comment:
this test is essentially still the same, but the old code was relying on the fact that if no offerings could be created, we'd return empty offerings in `OfferingsFactory`. Now it returns `nil` instead.